app/models/comment.rb

```diff
@@ -1,2 +1,10 @@
 class Comment < ApplicationRecord
+  belongs_to :user, class_name: 'User'
+  belongs_to :post, class_name: 'Post'
+
+  after_save :update_comments
+
+  def update_comments
+    post.increment!(:comments_counter)
+  end
```

Comment:
- [ OPTIONAL ] Great job for using one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the #update_comments method every time a new comment has been saved to database, but please I recommend making the update_comments method private ✨🚀🔥